colorbleed/plugins/maya/publish/validate_renderlayer_aovs.py

```diff
@@ -0,0 +1,60 @@
+import pyblish.api
+
+from avalon import io, api
+import colorbleed.api
+
+
+class ValidateRenderLayerAOVs(pyblish.api.InstancePlugin):
+    """Validate created AOVs / RenderElement is registered in the database
+
+    Each render element is registered as a subset which is formatted based on
+    the render layer and the render element, example:
+
+        <render layer>.<render element>
+
+    This translates to something like this:
+
+        CHAR.diffuse
+
+    This check is only used when the Extend Frames in 'renderGlobalsDefault'
+    is toggled on.
+
+    This check is needed to ensure the render output is still complete
+
+    """
+
+    order = pyblish.api.ValidatorOrder + 0.1
+    label = "Render Passes / AOVs Are Registered"
+    hosts = ["maya"]
+    families = ["colorbleed.renderlayer"]
+    actions = [colorbleed.api.SelectInvalidAction]
+
+    def process(self, instance):
+        invalid = self.get_invalid(instance)
+        if invalid:
+            raise RuntimeError("Found unregistered subsets: {}".format(invalid))
+
+    def get_invalid(self, instance):
+
+        invalid = []
+
+        asset_name = api.Session["AVALON_ASSET"]
```

Comment:
This is actually incorrect - the asset that will be published should be present inside the instance upon Collection - as such it should actually use the `instance.data["asset"]` or whatever the key is called, right?